cmd/kubelet/app/server.go

```diff
@@ -1107,7 +1107,7 @@ func RunKubelet(kubeServer *options.KubeletServer, kubeDeps *kubelet.Dependencie
 	}
 	podCfg := kubeDeps.PodConfig
 
-	rlimit.RlimitNumFiles(uint64(kubeServer.MaxOpenFiles))
+	rlimit.SetNumFiles(uint64(kubeServer.MaxOpenFiles))
```

Comment:
... shouldn't this be checking an error?

Reply:
Seems like the branch was behind `master` for a while. Should be updated now 🙇‍♂️